test/extended/deployments/deployments.go

```diff
@@ -242,6 +243,31 @@ var _ = g.Describe("deploymentconfigs", func() {
 		})
 	})
 
+	g.Describe("with multiple image change triggers", func() {
+		g.AfterEach(func() {
+			failureTrap(oc, "logging-kibana", g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should run a successful deployment [Conformance]", func() {
+			_, err := oc.Run("create").AsAdmin().Args("-f", loggingDeployerFixture).Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+			_, err = oc.Run("new-app").AsAdmin().Args("logging-deployer-account-template").Output()
```

Comment:
@deads2k @mfojtik I cannot seem to create cluster scoped resources with `AsAdmin`. Do we run other extended tests where we test cluster-scoped stuff?